pages/common/git-show-index.md

```diff
@@ -0,0 +1,12 @@
+# git show-index
+
+> Show the packed archive index of a Git repository.
+> More information: <https://git-scm.com/docs/git-show-index>.
+
+- Read a `.idx` file for a Git packfile and dump its contents to stdout:
```

Comment:
I think many will read this as `a I D X file`, instead of `a dot I D X file`. How about just
```suggestion
- Read an IDX file for a Git packfile and dump its contents to stdout:
```